python/katana/example_data.py

```diff
@@ -11,8 +11,15 @@
 __all__ = ["get_input", "get_input_as_url"]
 
 
-def get_inputs_directory(*, invalidate=False) -> Path:
+def get_inputs_directory(*, invalidate=False, rel_path=None) -> Path:
     inputs_dir = None
+    if rel_path == "external/csv-datasets":
+        location = os.environ['HOME']
+        for root, dirs, files in os.walk(location):
+            for d in dirs:
+                if d == "katana-enterprise":
+                    return Path(root) / d
```

Comment:
We really cannot do this. If it needs to be local, then you need to download it somewhere and return that URL, we cannot assume the source is checked out somewhere in the home directory.

Reply:
I updated the description. Hope it helps.
These csv files are already present in katana-enterprise/external directory and I'm trying to fetch that path. I'm following the current implementation seen in katana-enterprise/python/test/remote_test.py and other files:

dataset_dir = pathlib.Path(__file__).parent.parent.parent / "external/csv-datasets"

Reply:
since there is mapping for the cloud storage, maybe simply create a mapping for the local dirs and then pick one of the two